### DISCUSSION TITLE
Can this be used on Windows 10 to get rounded corners for application windows?

### DISCUSSION THREAD

**Discussion by Asad-13** [MicaForEveryone/MicaForEveryone#392] [Q&A]
If I am not wrong, this app can let us choose between Windows 7 style, Windows 11 style, and Windows 10 style corners (or that was ExplorerPatcher and I confused the two. Anyway, assuming it was this program)

Can I use it just to get rounded corners? I am a bit worried because I reckon it just might break some things, and don't want to deal with that right now.

**Comment by dongle-the-gadget** [comment]
No, selecting between square and rounded corners are Windows 11 only, and we don't have plans to support that option in Windows 10.
